canvas/templates/canvas/event_create.html

```diff
@@ -1,7 +1,12 @@
 {% extends 'base.html' %}
 {% load static %}
 
+{% block style %}
+    <link href="{% static 'css/jquery.datetimepicker.min.css' %}" rel="stylesheet">
+{% endblock %}
+
 {% block script %}
+    <script src="{% static 'js/jquery.datetimepicker.full.min.js' %}"></script>
```

Comment:
These static files should be in the global static folder. Please put them in static/datetimepicker/[version]/[js|css].
for example static/datetimepicker/2.3.7/js/datetimepicker.js